tests/tracking/fluent/test_fluent_autolog.py

```diff
@@ -0,0 +1,150 @@
+import pytest
+from unittest import mock
+import wrapt
+import inspect
+
+import mlflow
+
+import tensorflow
+import keras
+import fastai
+import sklearn
+import xgboost
+import lightgbm
+import mxnet.gluon
+import pyspark
+
+library_to_mlflow_module_without_pyspark = {
+    tensorflow: "tensorflow",
+    keras: "keras",
+    fastai: "fastai",
+    sklearn: "sklearn",
+    xgboost: "xgboost",
+    lightgbm: "lightgbm",
+    mxnet.gluon: "gluon",
+}
+
+library_to_mlflow_module = {**library_to_mlflow_module_without_pyspark, "pyspark": "spark"}
```

Comment:
Just curious why `pyspark` here is a string, not a module.

Reply:
i'll take a look at this and fix in a follow-up PR today. thanks for the comment, we missed this